extensions/vertx-http/runtime/src/main/java/io/quarkus/vertx/http/runtime/cors/CORSConfig.java

```diff
@@ -6,42 +6,37 @@
 
 import io.quarkus.runtime.annotations.ConfigGroup;
 import io.quarkus.runtime.annotations.ConfigItem;
-import io.vertx.core.http.HttpMethod;
 
 @ConfigGroup
 public class CORSConfig {
 
     /**
      * Origins allowed for CORS
      *
-     * Comma separated list of valid URLs. ex: http://www.quarkus.io,http://localhost:3000
-     * The filter allows any origin if this is not set.
+     * Comma separated list of valid URLs. ex: http://www.quarkus.io,http://localhost:3000 The filter allows any origin if this is not set.
      *
      * default: returns any requested origin as valid
      */
-    @ConfigItem
+    @ConfigItem(defaultValue = "*")
     public List<String> origins;
 
     /**
      * HTTP methods allowed for CORS
      *
-     * Comma separated list of valid methods. ex: GET,PUT,POST
-     * The filter allows any method if this is not set.
+     * Comma separated list of valid methods. ex: GET,PUT,POST The filter allows any method if this is not set.
      *
      * default: returns any requested method as valid
      */
-    @ConfigItem
-    public List<HttpMethod> methods;
+    public List<String> methods;
```

Comment:
Why this change?

Reply:
We can't use an Enum here, when we want to support * wildcard for methods as well.